docs/features/staticpages.rst

```diff
@@ -14,11 +14,66 @@ There are three types of static pages:
 #. Legal -- in addition to presenting content like a Regular page, Legal pages
    require that users agree to the content otherwise they are logged off the
    system.  Use these pages for presenting terms of service or changes in
-   licensing terms.
+   licensing terms that user must accept before they can use or continue to use
+   Pootle.
 #. Announcements -- these appear in the sidebar and can present special
-   instructions about projects.  Use the project code to ensure that this pages
-   appears as a sidebar for that project.  If you change these pages then they
-   will be presented to users when next they visit the project.
+   instructions about projects.  If you change these pages then they will be
+   presented to users when next they visit the project.  Announcement can also
+   make use of link rewriting to allow URLs to vary based on the language being
+   translated.
+
+
+Use **Admin -- Static Pages** to create and manage static pages.
 
 The static pages are by default formatted using HTML. But you can use Markdown
 or RestructuredText by setting :setting:`POOTLE_MARKUP_FILTER` correctly.
+
+
+Links in static pages
+---------------------
+
+When linking to a static page externally or in any customisations your links
+would be poiting to ``/pages/$slug``, such as ``/pages/gettting_started``.
+
+For linking to another static page from within a static page use the
+``#/$slug`` syntax.  Thus, if you created a *Getting Started* page as a static
+page which pointed to your *Licence Statement* legal page we'd use this
+``#/licence_statement`` in the URL.
+
+
+Special features of announcement pages
+--------------------------------------
+
+
+Naming your slug
+^^^^^^^^^^^^^^^^
+
+When creating an announcement page use a slug ``project/$project`` so that the
+page will be used on the ``$project`` project.
+
+Other slug names may be used:
+
+- ``project/$lang`` - for an announcement page that will appear on every single project
+  enabled for the ``$lang`` languages.
+- ``$lang/$project`` - for an announcement page that is specific to the
+  ``$project`` project in the ``$lang`` language.
+
+The prefered model though is to use the ``project/$project`` convention for a
+single easy to maintian page.
+
+
+Language link rewriting in Announcement pages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. pootle/apps/pootle_misc/templatetags/cleanhtml.py:rewrite_language_links()
```

Comment:
Anyway this doesn't feel like the go-to place for such comments. Also note the actual module path is prone to change.